k8s/distributed-chroma/values.yaml

```diff
@@ -50,6 +50,8 @@ logService:
   env:
     - name: OPTL_TRACING_ENDPOINT
       value: 'value: "jaeger:4317"'
+    - name: SYSDB_CONN
```

Comment:
when this PR merges will we have to update the log service config?

Reply:
Yes. Do you mean in the staging?